backend/loi-service/src/main/java/org/egov/works/web/models/LetterOfIndent.java

```diff
@@ -71,15 +71,22 @@ public class LetterOfIndent {
 
     @JsonProperty("oicId")
     private UUID oicId = null;
+
     @JsonProperty("status")
     private StatusEnum status = null;
+
     @JsonProperty("letterStatus")
     private String letterStatus = null;
+
     @JsonProperty("auditDetails")
     private AuditDetails auditDetails = null;
+
     @JsonProperty("additionalDetails")
     private Object additionalDetails = null;
 
+    @JsonProperty("bankGaurante")
+    private Object bankGaurante = null;

```

Comment:
Check the updated name from the loi-service branch. 
I think the type is String. 
You will get all the changes when merge those changes with your code.